build/configs/qemu/tc_64k/defconfig

```diff
@@ -783,26 +868,20 @@ CONFIG_TASH_CMDTASK_PRIORITY=100
 # CONFIG_SYSTEM_INIFILE is not set
 CONFIG_SYSTEM_PREAPP_INIT=y
 CONFIG_SYSTEM_PREAPP_STACKSIZE=2048
-# CONFIG_SYSTEM_INSTALL is not set
 # CONFIG_SYSTEM_RAMTEST is not set
 # CONFIG_SYSTEM_READLINE is not set
 CONFIG_SYSTEM_INFORMATION=y
-CONFIG_SYSTEM_CMDS=y
-# CONFIG_FS_CMDS is not set
 # CONFIG_ENABLE_DATE is not set
 # CONFIG_ENABLE_ENV_GET is not set
 # CONFIG_ENABLE_ENV_SET is not set
 # CONFIG_ENABLE_ENV_UNSET is not set
 CONFIG_ENABLE_FREE=y
 # CONFIG_ENABLE_HEAPINFO is not set
-# CONFIG_HEAPINFO_USER_GROUP is not set
-# CONFIG_HEAPINFO_USER_GROUP_LIST is not set
-# CONFIG_ENABLE_IRQINFO is not set
 # CONFIG_ENABLE_KILL is not set
 # CONFIG_ENABLE_KILLALL is not set
-# CONFIG_ENABLE_PS is not set
+CONFIG_ENABLE_PS=y
 # CONFIG_ENABLE_STACKMONITOR is not set
-# CONFIG_ENABLE_UPTIME is not set
+CONFIG_ENABLE_UPTIME=y
```

Comment:
Tash is disabled and why to enable ps and uptime commands?

Reply:
As @yashwanth686007 said, let's disable them by default.

Reply:
modify